src/extras-openmoji/healthcare/E306.svg

```diff
@@ -0,0 +1,21 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This emoji is currently listed as 'patch', but this is a bandage, bandaid or plaster. A patch is closer to an eye patch that a pirate wears or something. I'm changing it's name to bandage.